src/MagicWormhole.hs

```diff
@@ -59,6 +59,7 @@ module MagicWormhole
   , Peer.withEncryptedConnection
   , ClientProtocol.Connection
   , Peer.EncryptedConnection
+  , Peer.deriveKey
```

Comment:
Huh, I don't think I appreciated this Haskell feature for exposing APIs before just now.

So the top-level MagicWormhole module is exposing all kinds of stuff from deeper within the module hierarchy... And doing so by referencing names which _later_ get imported into this module...  Oookay...

Given

```
module MagicWormhole (
    Peer.deriveKey
) where
```

does that mean I can `import MagicWormhole (deriveKey)` or `import MagicWormhole (Peer)` or `import MagicWormhole.Peer (deriveKey)`?  (rhetorical question probably, I'll go read the import docs again, more carefully this time).

Reply:
The first one. `deriveKey` is now the function exposed out of `MagicWormhole` top level module.